frontend/src/metabase/admin/settings/components/widgets/PublicLinksListing/PublicLinksListing.jsx

```diff
@@ -135,7 +135,9 @@ export const PublicLinksDashboardListing = () => (
     revoke={DashboardApi.deletePublicLink}
     type={t`Public Dashboard Listing`}
     getUrl={dashboard => Urls.dashboard(dashboard)}
-    getPublicUrl={({ public_uuid }) => Urls.publicDashboard(public_uuid)}
+    getPublicUrl={({ public_uuid }) =>
+      Urls.publicDashboard({ uuid: public_uuid })
```

Comment:
Commented in `frontend/src/metabase/lib/urls/dashboards.ts`. I think we could preserve the current function API.